_sources/Conditionals/week3a1.rst

```diff
@@ -95,7 +95,7 @@ Chapter Assessment
     :autograde: unittest
     :practice: T
 
-    Write code that counts the number of words in ``sentence`` that contain *either* an "a" or an "e". Store the result in the variable ``num_a_or_e``.
+    Write code that counts the number of words in ``sentence`` that contains at least one of the letters "a" and "e". Store the result in the variable ``num_a_or_e``.
```

Comment:
not sure if this change makes it clearer, so probably not worth submitting upstream